Projects/Backendless.NET.Framework4.0/Backendless.NET.Framework4.0.csproj

```diff
@@ -30,6 +30,7 @@
     <DefineConstants>TRACE;PURE_CLIENT_LIB;NET_40;WITHRT</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
+    <Prefer32Bit>false</Prefer32Bit>
```

Comment:
what does it do?

Reply:
A .NET application compiled to x86 will fail to run on an ARM Windows system, but an “Any CPU 32-bit preferred” application will run successfully.

Reply:
But my mistake is that I left the parameter as "False".
I will change it to True

Reply:
Also, i can just delete this